.github/workflows/abtesting.yml

```diff
@@ -28,18 +28,33 @@ jobs:
         uses: actions/checkout@master
       - name: Setup
         run: |
-          cd abtesting
+          cd abtesting/LegacyABTestingQuickstart
           gem install bundler
           bundle install
           gem install xcpretty
           bundle exec pod install --repo-update
+          cd ..
           ../scripts/install_prereqs/abtesting.sh
         env:
           SAMPLE: ABTesting
           secrets_passphrase: ${{ secrets.GHASECRETSGPGPASSPHRASE1 }}
+          LEGACY: true
       - name: Build Swift
         run: |
           ./scripts/test.sh
         env:
-          SWIFT_SUFFIX: ''
+          SWIFT_SUFFIX: 'Swift'
```

Comment:
While confusing, the A/B testing sample doesn't need the Swift suffix to run the Swift build because the only build target present is a Swift target.

Reply:
Done ✅